tools/bin/hawqpylib/hawqlib.py

```diff
@@ -203,6 +203,13 @@ def check_hostname_equal(remote_host, user = ""):
     cmd = "hostname"
     result_local, local_hostname, stderr_remote  = local_ssh_output(cmd)
     result_remote, remote_hostname, stderr_remote = remote_ssh_output(cmd, remote_host, user)
+    if result_remote != 0:
+        print stderr_remote
+        print "Execute remote command failed on %s." % remote_host
```

Comment:
" with return code %d"  % result_remote?

Do we have error log or output in log file or hawq utility output?